### PR TITLE
[1.1.1.1] Introduce CLI section in Linux set up doc

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-9804

### CODE REVIEW

File: content/1.1.1.1/setup/linux.md [hunk 1]
```diff
@@ -4,13 +4,50 @@ pcx_content_type: how-to
...
-## GNOME (Ubuntu, Fedora, Pop!_OS etc)
+Consider the sections below to set up 1.1.1.1 using either the [command line interface (CLI)](#use-command-line-interface-cli) or a [graphical user interface (GUI)](#use-graphical-user-interface-gui) of your preference.
+
+## Use command line interface (CLI)
+
+Choose whether you want to use 1.1.1.1 or 1.1.1.1 For Families, and replace `1.1.1.1` with the corresponding [IPv4 or IPv6 address](/1.1.1.1/ip-addresses/) accordingly.
+
+### `resolv.conf`
+
+Usually, `etc/resolv.conf` is where you can configure the resolver IPs that your system is using.
```
Comment (vendemiat) [inline content/1.1.1.1/setup/linux.md]: leading '/' missing

File: content/1.1.1.1/setup/linux.md [hunk 2]
```diff
@@ -4,13 +4,50 @@ pcx_content_type: how-to
...
+
+```
+echo nameserver 1.1.1.1 | sudo tee /etc/resolv.conf
+```
+
+{{<Aside type="warning">}}
+Note that other systems, such as dynamic host configuration protocol (DHCP), may automatically write to `etc/resolv.conf` and change that configuration. In those cases, consider changing your network settings or DHCP to use 1.1.1.1
+{{</Aside>}}
+
+Alternatively, you can use `nano` to manually edit the file.
```
Comment (vendemiat) [inline content/1.1.1.1/setup/linux.md]: you can use your preferred editor

File: content/1.1.1.1/setup/linux.md [hunk 3]
```diff
@@ -4,13 +4,50 @@ pcx_content_type: how-to
...
+
+### `systemd-resolved`
+
+If you use `systemd-resolved` utility and the resolver IPs configuration is in `/etc/systemd/resolved.conf`, do the following:
+
+1. Run `sudo nano /etc/systemd/resolved.conf`
```
Comment (vendemiat) [inline content/1.1.1.1/setup/linux.md]: sudo $EDITOR, with a note to replace $EDITOR with preferred editor

File: content/1.1.1.1/setup/linux.md [hunk 4]
```diff
@@ -4,13 +4,50 @@ pcx_content_type: how-to
...
+Choose whether you want to use 1.1.1.1 or 1.1.1.1 For Families, and replace `1.1.1.1` with the corresponding [IPv4 or IPv6 address](/1.1.1.1/ip-addresses/) accordingly.
+
+### `resolv.conf`
+
+Usually, `etc/resolv.conf` is where you can configure the resolver IPs that your system is using.
+
+In that case, you can use the following one-line command to specify 1.1.1.1 as your DNS resolver:
+
+```
+echo nameserver 1.1.1.1 | sudo tee /etc/resolv.conf
```
Comment (hunts) [inline content/1.1.1.1/setup/linux.md]: Let's add a backup nameserver:
```
echo "nameserver 1.1.1.1\nnameserver 1.0.0.1" | sudo tee /etc/resolv.conf
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by RebeccaTamachiro** [comment]
Closes #12161

**Pull request merged**